client/app/src/js/controllers/landing.js

```diff
@@ -5,7 +5,17 @@
  */
 
 function LandingController(Screenshots, Auth) {
-
+  this.screenshots = [];
+  this.getScreenshots = function() {
```

Comment:
Check with Jackson -- are we resolving `getScreenshots` before we load this view?